### ISSUE TITLE
User login without password

### ISSUE THREAD

**Issue by lamdhoang** [meeting-room-booking-system/mrbs-code#3529]
Hi. I would like to have users log in to the system and make bookings without typing any password (only username is fine). Is it possible?

My configurations:
- MRBS version: MRBS 1.11.4
- database: mariadb-1:11.1.2
- php: 8.1.20
- system: linux-ubuntu 20.04.1

**Comment by campbell-m** [comment]
Do you want them to be able to make a booking without typing a username either?  If so, this is possible by setting

~~~php
$auth['allow_anonymous_booking'] = true;
~~~

Or do you want users to be able to have no password?  In other words you want users to appear in the Users table and have display names and email addresses, but just no password?

Or do you want users to be able to type any username when logging in, but not appear in the Users table?

**Comment by lamdhoang** [comment]
"''''
Or do you want users to be able to have no password? In other words you want users to appear in the Users table and have display names and email addresses, but just no password?

Or do you want users to be able to type any username when logging in, but not appear in the Users table?
''''''''

These are what I am looking for. But at this moment I only want users to log in without typing any password. It would be great if the users are the ones I created as an admin

**Comment by campbell-m** [comment]
No, sorry, that's not possible at the moment.  It would be an enhancement.

In the meantime you could just give them a simple password, eg a one character password.

**Comment by campbell-m** [comment]
No, sorry, as I said, that's not possible at the moment. It would be an enhancement.

Can you not just give everyone the same simple password?

**Comment by lamdhoang** [comment]
This is a request from my user, so simple password is the last option, how can I raise an enhancement request?

**Comment by campbell-m** [comment]
I've changed the label to enhancement.

**Comment by lamdhoang** [comment]
Thank you so much! Let me know if you have any updates.

**Comment by campbell-m** [comment]
There's a long list of enhancement requests so this one is unlikely to get looked at for a while I'm afraid.
